.ci/azure/mac.yml

```diff
@@ -1,229 +0,0 @@
-trigger:
-  branches:
-    include:
-    - master
-    - 'releases/*'
-  paths:
-    exclude:
-    - '*/docs/*'
-    - 'docs/*'
-    - '*/*.md'
-    - '*.md'
-    - '*/layer_tests_summary/*'
-    - '*/conformance/*'
-    - 'tests/layer_tests/*'
-
-pr:
```

Comment:
It is rather strange: there is a `pr` trigger yet I do not see this pipeline running on PRs and only in post-commit.